### DISCUSSION TITLE
Add the ability to move issues from one repo to a repo in a different organization

### DISCUSSION THREAD

**Discussion by nkolev92** [community/community#23293] [New to GitHub]
<p>The ability to move issues from one repo to another repo in the same organization is invaluable. <br>
It is very helpful for large teams, working cross repos.</p>
<p>Today it’s very common that certain projects often change organizations. <br>
Sometimes it’s acquisitions, sometimes restructuring.</p>
<p>Other times, an issue gets filled and it’s simply not part of this product. <br>
It’d be nice to be able to move such issues to the correct repo.</p>
<p>If there are concerns with issues getting moved around too much, I think it’s valid to require a certain permission level for the mover. </p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/nkolev92">@nkolev92</a>, thanks for getting in touch with us and for providing your feedback. Our product team is keen to hear this sort of feedback and we encourage you to share it with them here:</p>
<p><a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
<p>We can’t guarantee a response to every request, but we do appreciate the feedback! </p>
